doc/feature/compartmentReport.md

```diff
@@ -0,0 +1,422 @@
+Additions to compartmentReport to support ZeroEQ streams
+============
+
+## Motivation
+
+The current CompartmentReport lacks an implementation based on ZeroEQ as the
+SpikeReport does. There is a network plugin implementated with codash, but it
+has been deprecated. Since there's a new specification of the SpikeReport API
+ready for implementation, it makes sense to review the CompartmentReport
+interface to make it consistent with the new SpikeReport and prepare it for the
+ZeroEQ restrictions at the same time.
+
+This specification proposes a new interface for the CompartmentReport with
+these goals in mind.
+
+## API
```

Comment:
To me, a spec describes a new feature. In the case of the SpikeReport that was a completely new API. In case of this, it feels more like a delta and the current form makes it hard to understand what changes. (eg http://eyescale.github.io/equalizergraphics.com/documents/design/roi.html)

If I have no prior knowledge, I can read the full API doc in the .h.